src/main/java/seedu/duke/commands/GenerateSpecificDifficultyCommand.java

```diff
@@ -0,0 +1,46 @@
+package seedu.duke.commands;
+
+import seedu.duke.Utility.DifficultyLevel;
+import seedu.duke.errors.DukeError;
+import seedu.duke.exercisegenerator.GenerateExercise;
+import seedu.duke.exersisedata.ExerciseData;
+import seedu.duke.ui.Ui;
+
+import java.util.ArrayList;
+
+public class GenerateSpecificDifficultyCommand extends Command {
+    private final String difficulty;
+    private final String EASY = "easy";
+    private final String MEDIUM = "medium";
+    private final String HARD = "hard";
+
+    private final String BEGINNER = "beginner";
+    private final String INTERMEDIATE = "intermediate";
+    private final String EXPERT = "expert";
```

Comment:
Checkstyle: Names must match pattern '^[a-z][a-zA-Z0-9]*$'
Change to ```private static final String EASY = "easy"``` and shift above line 12